packages/wonder-blocks-modal/components/modal-launcher.md

```diff
@@ -14,9 +14,49 @@ const styles = StyleSheet.create({
     title: {
         marginBottom: 16,
     },
+
+    modalContent: {
+        margin: "0 auto",
+        marginTop: 40,
+        marginBottom: 40,
+        paddingLeft: 32,
+        paddingRight: 32,
+        maxWidth: 544,
```

Comment:
Not necessary at the moment, but in the future we may want to encapsulate common margins/paddings into some layout components than can easily be composed with the modals.